rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,20 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+references:
+    - https://attack.mitre.org/techniques/T1553/004/
```

Comment:
Could you please put the attack technique [tag](https://github.com/Neo23x0/sigma/wiki/Rule-Creation-Guide#tags) instead of the link in the references? 
Here is an example of the tag with a sub-technique:

https://github.com/Neo23x0/sigma/blob/c56cd2dfff6343f3694ef4fd606a305415599737/rules/windows/builtin/win_alert_mimikatz_keywords.yml#L14